types/selenium-webdriver/lib/virtual_authenticator.d.ts

```diff
@@ -0,0 +1,108 @@
+/**
+ * Protocol for virtual authenticators
+ * @enum {string}
+ */
+export interface IProtocol {
+    CTAP2: string;
+    U2F: string;
+}
+
+/**
+ * AuthenticatorTransport values
+ * @enum {string}
+ */
+export interface ITransport {
+    BLE: string;
+    USB: string;
+    NFC: string;
+    INTERNAL: string;
+}
+
+export const Protocol: IProtocol;
+export const Transport: ITransport;
```

Comment:
```suggestion
/**
 * Protocol for virtual authenticators
 * @enum {string}
 */
export enum Protocol {
    CTAP2 = 'ctap2',
    U2F = 'ctap1/u2f'
}

/**
 * AuthenticatorTransport values
 * @enum {string}
 */
export enum Transport {
    BLE = 'ble',
    USB = 'usb',
    NFC = 'nfc',
    INTERNAL = 'internal',
}
```

These look like they're meant to be used as enums

Reply:
Change incorporated.